tests/database/postgres/test_query_factory.py

```diff
@@ -65,6 +65,12 @@ def int_fake_column_generator():
         get_data_type=Mock(return_value=FakeDataType.INT),
         get_value=Mock(return_value=645)
     )
+@pytest.fixture
+def uuid_fake_column_generator():
+    return Mock(
+        get_data_type=Mock(return_value="UUID"),
```

Comment:
I'm not sure mocking the return value to a string is a good idea here. See my comments on
https://github.com/jerometwell/pynonymizer/pull/27/files#diff-f7683774162643a2e41307342703774c92817ef9a02c0fb58f21c0bfd6cfde00R65